Sources/PostgreSQL/Connection/PostgreSQLConnection.swift

```diff
@@ -122,11 +122,11 @@ public final class PostgreSQLConnection: DatabaseConnection, BasicWorker, Databa
 
     /// Closes this client.
     public func close() {
-        _ = executeCloseHandlersThenClose()
+        _ = _executeCloseHandlersThenClose()
     }
 
     /// Executes close handlers before closing.
-    private  func executeCloseHandlersThenClose() -> Future<Void> {
+    public func _executeCloseHandlersThenClose() -> Future<Void> {
```

Comment:
Mention in a comment that this is only suggested to be used in tests? Or make it `internal`, so that it can be used with `@testable import`?